desktop-src/Controls/lvm-setgroupinfo.md

```diff
@@ -28,19 +28,19 @@ Sets group information.
 </dt> <dd>ID that specifies the group whose information is to be set.</dd> <dt>
 
 *lParam* 
-</dt> <dd>Pointer to an <a href="/windows/desktop/api/Commctrl/ns-commctrl-taglvgroup">**LVGROUP**</a> structure that contains the information to set.</dd> </dl>
+</dt> <dd>Pointer to an <a href="https://docs.microsoft.com/en-us/windows/win32/api/commctrl/ns-commctrl-lvgroup">LVGROUP</a> structure that contains the information to set.</dd> </dl>
 
 ## Return value
 
 Returns the ID of the group if successful, or -1 otherwise.
 
 ## Remarks
 
+<b>LVGROUP.mask</b> should not include the <b>LVGF_GROUPID</b> flag when updating fields such as <b>pszTask</b> or <b>uAlign</b> on an existing group.
```

Comment:
Sounds too broad for me, you can include LVGF_GROUPID as long as you actually change the ID. I think its better to document that this flag is a change-id request and will fail if the target group id already exists. How about something like
```suggestion
Including <b>LVGF_GROUPID</b> in <b>LVGROUP.mask</b> implies changing the group ID and will fail if the ID given in <b>LVGROUP.iGroupId</b> is already used. When updating fields such as <b>pszTask</b> or <b>uAlign</b> on an existing group including <b>LVGF_GROUPID</b> in <b>LVGROUP.mask</b> is not required.
```

Reply:
How about something like this?
```suggestion
To change a group ID of an existing group add <b>LVGF_GROUPID</b> to <b>LVGROUP.mask</b> and set <b>LVGROUP.iGroupId</b> to the new ID. The call will fail if <b>LVGROUP.iGroupId</b> contains ID of an existing group.<br />
To update other properties of an existing group (e.g. update an alignment of the header or footer text for the group, <b>uAlign</b>) <b>LVGROUP.mask</b> must not contain <b>LVGF_GROUPID</b>, else the update will fail.
```